Example/UI Examples/BrowseViewController.swift

```diff
@@ -0,0 +1,179 @@
+//
+//  BrowseViewController.swift
+//  UI Examples
+//
+//  Created by Ben Guo on 7/18/17.
+//  Copyright © 2017 Stripe. All rights reserved.
+//
+
+import UIKit
+import Stripe
+
+class BrowseViewController: UITableViewController, STPAddCardViewControllerDelegate, STPPaymentMethodsViewControllerDelegate, STPShippingAddressViewControllerDelegate {
+
+    enum Demo: String {
+        static let count = 5
+        case STPPaymentCardTextField = "Card Field"
+        case STPAddCardViewController = "Card Form"
+        case STPPaymentMethodsViewController = "Payment Methods"
+        case STPShippingInfoViewController = "Shipping Info"
+        case ChangeTheme = "Change Theme"
+
+        init?(row: Int) {
+            switch row {
+            case 0: self = .STPPaymentCardTextField
+            case 1: self = .STPAddCardViewController
+            case 2: self = .STPPaymentMethodsViewController
+            case 3: self = .STPShippingInfoViewController
+            case 4: self = .ChangeTheme
+            default: return nil
+            }
+        }
+    }
```

Comment:
Hmm I think I would reverse this definition so that the enum is on Integer and the String is a method:

```swift
enum DemoTwo: Int {
    case STPPaymentCardTextField
    case STPAddCardViewController
    case STPPaymentMethodsViewController
    case STPShippingInfoViewController
    case ChangeTheme

    func title() -> String {
        switch self {
        case .STPPaymentCardTextField:
            return "Card Field"
        ...
    }
}
```

one thing I'm alluding to here is maybe we should detail labels to the rows that indicate the class name

```
Card Field
-> STPPaymentCardTextField

Card Form
-> STPAddCardViewController
...
```